website_sale_line_total/views/website_sale_line_total_view.xml

```diff
@@ -27,4 +27,39 @@
         </xpath>
     </template>
 
+    <template id="add_payment_item_line_total" inherit_id="website_sale.payment">
+        <xpath expr="//table[@id='cart_products']/tbody/tr/td[span[contains(@t-field,'line.price_reduce_tax')]]" position="attributes">
+            <attribute name="id">td-unit_price</attribute>
+        </xpath>
+        <xpath expr="//table[@id='cart_products']/tbody/tr/td[div[contains(@t-esc,'line.product_uom_qty')]]" position="attributes">
+            <attribute name="id">td-qty</attribute>
+        </xpath>
+
+        <xpath expr="//table[@id='cart_products']/thead/tr/th[@width='120']" position="before">
```

Comment:
this is a poor selector IMO: are you selecting the one containing "Quantity"?
If yes: either you select by position or by text content.